### DISCUSSION TITLE
10% Early Bird discount for 4-day Security Onion 2 Fundamentals for Analysts and Admins Training Class

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#3263] [Announcements]
https://blog.securityonion.net/2021/03/10-early-bird-discount-for-4-day.html

![Screen Shot 2021-03-03 at 12 36 09 PM](https://user-images.githubusercontent.com/1659467/109849773-c3753180-7c1f-11eb-9c3a-b02cdc87a1f8.png)

**Comment by TOoSmOotH** [comment]
Bumping this to the top

**Comment by dougburks** [comment]
Only 2 months left to register for this training class!
https://securityonionsolutions.com/livetraining
